models/coreModels/Email.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+
+const emailSchema = new mongoose.Schema({
+  emailKey: {
+    type: String,
+    unique: true,
+    lowercase: true,
+    required: true,
+  },
+  emailName: {
+    type: String,
+    unique:true,
+    required:true,
+  },
+  emailVariables:{
+    type:Array
+  },
+  emailBody: {
+    type: String,
+    required: true,
+  },
+  emailSubject: {
+    type: mongoose.Schema.Types.Mixed,
```

Comment:
what's reason to use mongoose.Schema.Types.Mixed,
type: String, is could be okay